### DISCUSSION TITLE
Subscription canceling and refund

### DISCUSSION THREAD

**Discussion by WaelHarrath** [community/community#34299] [Copilot Conversations]
I am using github copilot for the first time , after the trial peiod has ended i forgot to cancel the subscription and my boss card got charged for yearly subscription , so my question is how can i cancel this yearly subscription and is there a refund when canceling ?

**Comment by iamcjwilliams** [comment]
@WaelHarrath Visit here to [cancel your subscription](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription#canceling-your-github-copilot-subscription). To receive a refund you, please open a ticket on our [Support page](https://support.github.com/contact).

**Reply by WaelHarrath** [reply]
@iamcjwilliams i canceled the subscription and opened a ticket to the support team but i didn't get any response until now

**Reply by yiyuxin** [reply]
Same here

**Reply by iamcjwilliams** [reply]
@WaelHarrath @yiyuxin  Our copilot team is currently experiencing a high volume of tickets, so responses will be delayed. Thank you for your patience

**Reply by quangtuan28200** [reply]
same, even i just subscribed for the trial yesterday and today i got the bill for 100$ for the year plan

**Reply by TheColorPlum** [reply]
Same here

**Reply by ghost** [reply]
same here

**Comment by mayankahuja2468** [comment]
Yup same. I too cancelled and requested for a refund as I am student and don't to pay 100$ for the subscription.

**Reply by Shubham07-bit** [reply]
have you got the refund?

**Comment by SannivaHalderBapon** [comment]
Exact same thing happened to me, and it's been deducting 10$ every month now that I checked it...... but probably we're not gonna be eligible for refunds...

**Comment by kerryj89** [comment]
1. https://support.github.com/contact/account
2. "Account or Billing Issue" link
3. "What do you need help with?" -> "Billing, payments, or receipts"
4. (This part I forget how exactly I triggered it, but I had "Refund" in the issue title... somehow it suggested refunding Copilot and I clicked on it)
5. "Virtual Agent" will ask if you want to cancel and get a refund, I said yes... it was painless.

**Reply by TheColorPlum** [reply]
^This should be higher. Worked for me

**Reply by kai-phan** [reply]
Its worked, many thanks

**Comment by pucha48** [comment]
Use Github Support with this keywords Virtual Assistant will kick in.


<img width="996" alt="Screenshot 2025-06-27 at 11 29 06 AM" src="https://github.com/user-attachments/assets/edb71e1e-1ba9-4c93-97fb-4b1892a0a78f" />

**Reply by duynguyen1205** [reply]
Did you get the refund now ?

**Reply by LeeNguyen1211** [reply]
> Did you get the refund now ?

Hi, may i ask if u got the refund? How long did it take?

**Comment by lxc-xx** [comment]
I was charged for 300 USD for copilot that I have never used. I guess applied that trial, and I totally forgot it and never used copilot.

June 12, two weeks ago, I opened a ticket, and applied refund through the Virtual Assistant

The subscription was canceled immediately but the refund was never received. The ticket was open and not answered.

**Reply by lxc-xx** [reply]
The ticked was #3477385

**Reply by jiahaoc1993** [reply]
Did you get the refund now ?

**Comment by ngn999** [comment]
Just open this: https://support.github.com/contact-next/product-selection/copilot/billing/billing-main/billing-cancellation

**Comment by mariusccna** [comment]
Really hope to get the refund, I really need the money at this point ( ticket #3884289)  -> I was billed an hour before the ticket was placed.

**Reply by delacruz-uge** [reply]
Have you got your refund?

**Comment by lxc-xx** [comment]
They only refund me 1 year fee. 2 year fee charged for zero usage.

**Reply by Shubham07-bit** [reply]
How did you got your refund?

**Reply by delacruz-uge** [reply]
How many days did it took you to get your refund?

**Reply by Shubham07-bit** [reply]
almost a month

**Comment by lxc-xx** [comment]
Complained and waited for many months.

**Comment by jtsai-quid** [comment]
Now use this link. https://support.github.com/contact-next/product-selection/copilot/copilot-billing
<img width="1545" height="1297" alt="螢幕擷取畫面 2026-03-21 204622" src="https://github.com/user-attachments/assets/af843641-b6b5-4e7b-be9d-48bee35adb1b" />

**Reply by sashitomo** [reply]
Have you got your refund?

**Reply by sashitomo** [reply]
> > 你收到退款了吗？
> > Have you received the refund? I already have the refund receipt, but there is no refund record in my bank account
> > 你收到退款了吗，我已经有了退款收据，可是银行卡里没退款记录

I finally got my refund, but no thanks to GitHub—I had to file a dispute with PayPal. GitHub went MIA for more than 15 days, so PayPal just gave me the money back. Honestly, if I hadn't gone through PayPal, I probably would've never seen that refund in my life.

**Comment by DarthMavenus** [comment]
I CAN´T BELIVE YOU PEOPLE on Github who developed copilot! My copilot claimed my Backend was all ready for deployment. 

Next thing I know when checking with Codex there are massive buggs in my main backend, could have cost me (Im an entreprenour with my own company) millions of dollars if I had started marketing. 

What the acctual FU** are you doing over there? I thought Copilot had it all down and correct!
